cpg_utils/workflows/inputs.py

```diff
@@ -161,6 +157,24 @@ def _populate_alignment_inputs(
     for sample in cohort.get_samples():
         for d in found_seqs_by_sid.get(sample.id, []):
             seq = Sequence.parse(d, check_existence=check_existence)
+            if only_batches:
+                if 'batch' not in seq.meta:
```

Comment:
Should probably come up with a list of these "patterns" around the place. Or even maybe define them in MM itself to make it clearer how these are used and provided.

Reply:
Agree, makes sense to do this in metamist!

Reply:
It also took me a few seconds to realize that this is about _sample_ batches and completely unrelated to Hail batches. The term "batch number" is unfortunately somewhat overloaded. This becomes even more tricky as we also want to store Hail Batch related information in Metamist, e.g. resource usage.

Can we make this a little clearer by using `sample_batch` or similar instead?